### ISSUE TITLE
Keyboard problems on "Messages" tab (web messenger)

### ISSUE THREAD

**Issue by rstruminski** [indywidualny/FaceSlim#176]
I'm not sire whether this is a Face Slim problem or a problem of the web messenger itself, but:

Whenever I try to write something my keyboard randomly disappears. Sometimes it shows up again by itself, sometimes it doesn't. And often it selects one or more of the last letters I typed, so that I overwrite them while trying to type the next word.

It basically makes the chat unusable to me. Any ideas?

**Comment by Nutomic** [comment]
I'm getting the same problem. Additionally, backspace sometimes duplicates the previous characters, instead of deleting them. All of this is extremely annoying.

**Comment by rstruminski** [comment]
Oh yes - forgot about the backspace thing. I have that, too.

**Comment by indywidualny** [comment]
Its not supposed to be working on a mobile device. I forced it. It's buggy
and somehow broken but I cannot do anything about it...

sob., 11.06.2016, 16:41 użytkownik rstruminski <notifications@github.com>
napisał:

**Comment by ellypsis** [comment]
Same problem for me.
So for now, there is no alternative to Messenger on Android except using the basic mode of Face Slim :s

**Comment by indywidualny** [comment]
There's a beta with a possibility to set your own user agent. This way it's possible to use "old" messages :) It's gonna be released as a stable version soon. No more messenger :)

**Comment by indywidualny** [comment]
https://github.com/indywidualny/FaceSlim/releases/tag/v2.6.0-beta3

Please check it out. If it's not buggy i will release it tomorrow.

**Comment by ellypsis** [comment]
Works fine for now. Thanks a lot!!

**Comment by indywidualny** [comment]
I hate Facebook so much I decided to fix this app because they broke it. No problem :)

**Comment by rstruminski** [comment]
Very nice, seems to work! Thanks!

**Comment by indywidualny** [comment]
Yep. It's supposed to. Someone may like web messenger so I left it. I removed it everywhere else. Just tap messages icon at the blue bar :)

**Comment by nlvivar** [comment]
I'm seeing this behavior now. It think it's been a week or so. In on v.2.9.0.
